### ISSUE TITLE
[med-high] up front provide overview of difference and timeline

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-intro-wcag#188]
from [r12a](https://github.com/w3c/team-strat/issues/139):

> Answers to most of these questions should be in the WAI website. We welcome input on making them more clear.

The introduction to the site says:

> The WCAG 3 documents explain how to make the web more accessible to people with disabilities. WCAG 3 applies to web content, apps, tools, publishing, and emerging technologies on the web.

That sounds the same as i'd expect to hear in an intro to WCAG2, and therefore my brain already begins to glaze over on the assumption that we've hit verbiage.  It would be useful to answer the question that i think is upper-most in people's minds when they come to the page: how is WCAG3 different from WCAG2, and what does it mean for the way i'm working (in one or two sentences).  This relates to my question above. _which is:_ may not be clear to the user as to whether this involves substantive changes about how to achieve accessibility? or is simply a repackaging of the same information in a way that is more accessible to the developer

It looks like you can find some information to address that question, but it's waaaay down the page and you have to read through lots of detailed bullet points to build the overview for yourself.

hope that helps.

**Comment by shawna-slh** [comment]
Thanks @r12a 

I added this to the summary at the top:

> WCAG 3 has similar fundamental and specific accessibility requirements. WCAG 3 has a different structure, different conformance model, and broader scope.

I'll close this for now, and welcome you or anyone to re-open it if you think that's not sufficient or have other suggestions.
